scripts/hudson/narayana.sh

```diff
@@ -468,15 +477,14 @@ function download_as {
   zip=$(ls wildfly-*-SNAPSHOT.zip) # example the current latest is wildfly-preview-27.0.0.Beta1-SNAPSHOT.zip
 
   export JBOSS_HOME=${JBOSS_HOME:-"${PWD}/${zip%.*}"}
+  echo "JBOSS_HOME=$JBOSS_HOME"
```

Comment:
I think this is tidy up too, perhaps to state earlier in the build what JBOSS_HOME is set to? That seems a good thing to do in a tidy up commit - were the "" required?

Reply:
The bash script runs with `set -x` so it just adds noise.

Reply:
I am not sure if I understand what you are responding to, I could imagine your response being why this type of line would be removed (rather than moved).

Reply:
If you do decide not to include this line here, then you might also want to keep your removal of the `echo JBOSS_HOME=$JBOSS_HOME` but if you do that should be in a commit that tidies up narayana.sh rather than connecting it to WildFly

Reply:
> I am not sure if I understand what you are responding to, I could imagine your response being why this type of line would be removed (rather than moved).

The `echo JBOSS_HOME=$JBOSS_HOME` statement is superfluous because the script runs with `set -x` which already logs the value and, moreover, it logs it at the point it was set.